tests/network/vsftpd.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: vsftpd QAM regression testsuite
+#          not testing SSL now due to https://bugzilla.suse.com/show_bug.cgi?id=1116571
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use base 'consoletest';
+use testapi;
+use strict;
+use lockapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console 'root-console';
+    if (check_var('HOSTNAME', 'server')) {
```

Comment:
I think this barrier can be replaced with simple `wait_for_children` call at the end